tracpro/msgs/tasks.py

```diff
@@ -55,40 +60,39 @@ def org_task(self, org):
         from .models import InboxMessage
         from tracpro.contacts.models import Contact
 
-        client = org.get_temba_client()
+        def update_or_create_message(message, org):
+            contact = Contact.objects.filter(uuid=message.contact.uuid).first()
+            # If the contact sync task hasn't gotten this contact yet,
+            # don't get the message yet
+            if contact:
+                InboxMessage.objects.update_or_create(
+                    rapidpro_message_id=message.id,
+                    org=org,
```

Comment:
I believe we want everything after rapidpro_message_id to be passed as the `defaults` dictionary here (https://docs.djangoproject.com/en/1.8/ref/models/querysets/#update-or-create).